apps/estuary/config/test.exs

```diff
@@ -1,6 +1,17 @@
 use Mix.Config
 
+config :logger, level: :warn
```

Comment:
This will likely get overridden btw.  Do not depend on this being set this way, per the arts and crafts thing i brought up.

Reply:
JMS. I think we're just doing this 1) to be consistent and 2) for warn when running just our tests, but I don't think we're relying on it.